sample/src/main/java/com/davidecirillo/multichoicesample/sampleToolbar/SampleToolbarActivity.java

```diff
@@ -90,7 +90,8 @@ private void setUpAdapter(List<String> strings) {
 
         MultiChoiceToolbar multiChoiceToolbar =
                 new MultiChoiceToolbar.Builder(SampleToolbarActivity.this, toolbar)
-                        .setTitles(toolbarTitle(), "item selected")
+                        .setTitles(toolbarTitle(), "item selected") // TODO crashing when removed !?
```

Comment:
Looks like there is an issue with the way i'm populating the data with RXjava check out that piece of code, it's just generating some items.